docs/content/3.guide/1.writing/6.csv.md

```diff
@@ -0,0 +1,43 @@
+---
+title: CSV
+description: 'Nuxt Content can query CSV formatted content.'
+---
+
+# CSV
+
+Nuxt Content can query CSV formatted content. The `body` of the output is an array containing every row as objects.
+
+## Example
+
+::code-group
+
+```yaml [content/hello.yml]
+title: "Hello Content v2!"
+description: "The writing experience for Nuxt 3",
+category: 'announcement'
+```
```

Comment:
```suggestion
```yaml [content/hello.csv]
title,description,category
Hello Content v2!,The writing experience for Nuxt 3,announcement
```
```